earth_enterprise/src/fusion/tools/gee_test/README

```diff
@@ -25,6 +25,7 @@ sudo /opt/google/bin/geconfigureassetroot --new --assetroot  /usr/local/google/g
 
 sudo /opt/google/bin/geselectassetroot --assetroot  /usr/local/google/gevol_test/assets
```

Comment:
I meant how do we decide to put an asset root under `/usr/local/` one time, `/gevol` at another, tutorial assets under `/opt/google/share`? I don't follow the logic there, and I'm not sure it conforms to [FHS](http://www.pathname.com/fhs/pub/fhs-2.3.html).